redis/src/main/scala/zio/redis/Input.scala

```diff
@@ -282,14 +282,18 @@ object Input {
       Chunk(encodeString("RETRYCOUNT"), encodeString(data.toString))
   }
 
-  case object XGroupCreateInput extends Input[XGroupCommand.Create] {
-    def encode(data: XGroupCommand.Create)(implicit codec: Codec): Chunk[RespValue.BulkString] = {
+  final case class XGroupCreateInput[K, G, I]()(implicit
```

Comment:
I think that we should delete the custom inputs for commands with 2 words name and add this command support how i've done in my [scripting PR ](https://github.com/zio/zio-redis/pull/269/files#diff-92c48a34b5fa9685e7534a627e2bd4ad087d9dfc7f8b171d13fdb12f3e1de8eb)
@mijicd your opinion for that thought is important. thanks)

Reply:
Sure, it might scrape some stuff.